draft-ietf-scitt-architecture.md

```diff
@@ -371,6 +371,157 @@ Issuers SHOULD use consistent decentralized identifiers for all their Statements
 They MAY update their DID manifest, for instance to refresh their signing keys or algorithms, but they SHOULD NOT remove or change any prior keys unless they intend to revoke all Signed Statements that are registered as Transparent Statements issued with those keys.
 This DID appears in the Issuer protected header of Signed Statements' Envelopes, while the version of the key from the manifest used to sign the Signed Statement is written in the `kid` header.
 
+`kid` MUST either be an absolute URL,
+or a relative URL. Relative URL MUST be
+relative to an `iss` value. When relative URL is used,
+`iss` MUST also be present in the protected header.
+
+Resolving `kid` MUST return an identity document of a registered content type (a set of public keys).
+In the case of `kid` being an absolute DID URL, the identity document is called a DID Document,
```

Comment:
nit: linebreak on sentences across the PR

Reply:
fix it editorial